app/controllers/votes_controller.rb

```diff
@@ -0,0 +1,25 @@
+class VotesController < ApplicationController
+  before_action :authenticate_request, except: [:index]
+
+  def index
+    render json: project.total_number_of_votes
```

Comment:
when the user upvotes or down votes it will kick off a request to this end point to get the up to date vote count. But now that I think about it, it would probably be better to include the vote count in the response of the create and delete actions instead.